### ISSUE TITLE
Are all the current functions supported in parse-server?

### ISSUE THREAD

**Issue by bagusflyer** [ParsePlatform/parse-server#64]
Just heard a good news said the **Push Notification Service** and **Dashboard** will be provided soon within months. But I'm still concerning on what I didn't know or what I didn't expect. A very simple example is the email verification. Apparently parse.com is using their smtp server to send html generated by the server. My question is if there is a place for us to config this kind of thing which we need not care when we use parse.com service.

Currently I'm in the middle of developing of an application in parse. My CMS, my mobile app are all relying on parse. To change to another backend server or write my own is very painful because 80% of works are done. The best solution I think by far is to continue to develop and maybe migrate to own host server with parse-server in half year. But I'm not very confident yet. I have some concerns like what I mentioned above.

In short, my question is actually are all the functions are supported in open-source parse-server? Or which are the functions are not supported? If I know this, I definitely can do some preparations first.

**Comment by Kinzi** [comment]
+1 especially what about: email verification, password reset, ect.

**Comment by davidaik** [comment]
+1 I'm wondering about Class Level Permissions too.

**Comment by JcMinarro** [comment]
+1

**Comment by Morbix** [comment]
+1 Dashboard turn Open-Source too

**Comment by gfosco** [comment]
Migrated class level permissions work, with the exception of 'pointer permissions' which was a recent new feature... None of the email/verification/reset stuff is implemented (have at it!) and we're doing our best to get the dashboard going soon.  I think these requests are covered in other issues, but please submit PRs if you would like to contribute.  Thanks.
